android/jni/com/mapswithme/maps/Framework.hpp

```diff
@@ -50,6 +48,9 @@ namespace android
 
     string m_searchQuery;
 
+    typedef map<gui::EWidget, gui::Position> TGuiPositions;
```

Comment:
We use typedefs only if needed - when they are reused more than once. In this case it is not needed.

> On Oct 8, 2015, at 14:44, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/jni/com/mapswithme/maps/Framework.hpp <https://github.com/mapsme/omim/pull/171#discussion_r41573243>:
> 
> > @@ -50,6 +48,9 @@ namespace android
> >  
> >      string m_searchQuery;
> >  
> > +    typedef map<gui::EWidget, gui::Position> TGuiPositions;
> For similarity with analogic declarations above.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/171/files#r41573243>.
>